packages/ui/origin-ui-core/src/components/certificates/Inbox/InboxItem.tsx

```diff
@@ -157,7 +160,12 @@ export function InboxItem(props: {
                                     onChange={() => onCertificateSelect(cert.id, device.id)}
                                 />
                             </div>
-                            <div className={classes.iconContainer}>
+                            <div
+                                className={classes.iconContainer}
+                                style={{
+                                    fill: isSelected ? PRIMARY_COLOR : '#969696'
```

Comment:
`#969696` could be replaced using `LightenColor()`

Reply:
done